README.md

```diff
@@ -1,11 +1,18 @@
-# curriculum-databases-projects-template
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-> This template should be used for database related projects at Microverse.
-> Generate your own repository, update this README and edit all files content while working on projects. You should not be adding any new files unless asked otherwise.
+# Vet_clinic Database - Create Animals Table
+
+> This README has detailed information on how to create <code>animals</code> table, INSERT data and SQL QUERY for <code>vet_clinic</code> database project.
 
 
 ## Getting Started
 
+### To launch the project locally:
+
+- Clone this repository in your local machine terminal:
+
+git clone <code>https://github.com/Okechukwu-muokwugwo/vet_clinic</code>
+
 This repository includes files with plain SQL that can be used to recreate a database:
 
 - Use [schema.sql](./schema.sql) to create all tables.
```

Comment:
- [ ] Kindly use the latest version of readme template. You can get it [here](https://github.com/microverseinc/readme-template/blob/master/README.md) 😊 You are using the older version of readme file but it would be nice if you would use the latest version of the readme file 👍